src/HealthChecks.Publisher.Datadog/DatadogPublisher.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.Extensions.Diagnostics.HealthChecks;
+using StatsdClient;
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace HealthChecks.Publisher.Datadog
+{
+    class DatadogPublisher
```

Comment:
I would like to add some unit tests for this class, but it looks like we only DependencyInjection and UI tests in the UnitTests project. What would be the best place to put these?
@unaizorrilla, @lurumad, @carloslanderas, @eiximenis